doc/rst/source/explain_symbols.rst_

```diff
@@ -97,52 +96,52 @@
         Upper-case versions **E**, **J**, and **W** are similar to **e**, **j** and **w**
         but expect geographic azimuths and distances.
 
-    **-Se**
+    **-Se**\ *direction*/*major_axis*/*minor_axis*
         **e**\ llipse. If not given, then *direction* (in degrees counter-clockwise from horizontal),
-        *major_axis*, and *minor_axis* must be found in columns 3, 4, and 5. This option yields
-        a Cartesian ellipse whose shape is unaffected by the map projection.  If only a *size* is
-        given then we plot a degenerate ellipse (circle) with given diameter.
+        *major_axis*, and *minor_axis* must be found after the location [and *value*] columns. This option yields
+        a Cartesian ellipse whose shape is unaffected by the map projection.  If only a single
+        *size* is given then we plot a degenerate ellipse (circle) with given diameter.
```

Comment:
> If only a single *size* is given then we plot a degenerate ellipse (circle) with given diameter.

I don't test it, but does it mean I can give only one *size* even in an input file?

Reply:
We should check.   I think **-Se-** or **-SE-** means read a single item, while **-Se**_diam_ or **-SE**_diam_ do no longer need that extra minus sign.

Reply:
These all work for degenerate geo ellipse

echo 0 0 300 | gmt plot -R-3/3/-3/3 -Jm1i -SE- -Gred -B -pdf t
echo 0 0 | gmt plot -R-3/3/-3/3 -Jm1i -SE-300 -Gred -B -pdf t
echo 0 0 | gmt plot -R-3/3/-3/3 -Jm1i -SE300 -Gred -B -pdf t

Reply:
I think you misunderstood what I say. The new documentation for **-Se** says:

>  **-Se**\ *direction*/*major_axis*/*minor_axis*
>        **e**\ llipse. If not given, then *direction* (in degrees counter-clockwise from horizontal),
>         *major_axis*, and *minor_axis* must be found after the *coordinates* [and *value*] columns. This option yields
>        a Cartesian ellipse whose shape is unaffected by the map projection.  If only a single
>        *size* is given then we plot a degenerate ellipse (circle) with given diameter.

I think the documentation means the following command works:
```
echo 0 0 | gmt plot -R-3/3/-3/3 -Jm1i -Se1c -Gred -B -pdf t
```
but users may think that the below command works (it doesn't):
```
echo 0 0 1c | gmt plot -R-3/3/-3/3 -Jm1i -Se -Gred -B -pdf t
```